app/models/image.rb

```diff
@@ -37,13 +38,13 @@ class Image < ApplicationRecord
 
   validates_attachment_presence :content
   validates_attachment_size :content, less_than: 20.megabytes
-  validates_attachment_content_type :content, content_type: %w(image/tiff image/jpeg image/jpg image/png image/x-png image/gif)
+  validates_attachment_content_type :content, content_type: %w[image/tiff image/jpeg image/jpg image/png image/x-png image/gif]
```

Comment:
Do we ever want to _not_ convert `image/tiff`? There's also `image/x-tiff` to consider, if we're supporting tiff.